.github/workflows/portal-loop-txs-exporter.yml

```diff
@@ -0,0 +1,46 @@
+name: Backup Portal Loop
+
+on:
+  # allow to run workflow manually
+  workflow_dispatch: { }
+
+  # Triggers the workflow every day at 1am
```

Comment:
Isn't this every hour?

Reply:
yes it is every hour, forgot to update the comment